### PR TITLE
Add "new" API Throttle code

### PR DESCRIPTION
**Pull request by tolbon**
https://developers.facebook.com/docs/graph-api/advanced/rate-limiting
At the end of this page you see list of rate limiting code.
 
I keep code "341" cause in an other page of the docs :
https://developers.facebook.com/docs/graph-api/using-graph-api#errors
We can see 341 code

### CODE REVIEW

**Comment by Nyholm** [comment]
This should also be merged to master.

**Comment by tolbon** [comment]
Do you want i open an another PR for Master ?
All new PR go to master or in this branch ?

File: src/Facebook/Exceptions/FacebookResponseException.php [hunk 1]
```diff
@@ -70,6 +70,7 @@ public static function create(FacebookResponse $response)
     {
         $data = $response->getDecodedBody();
 
+
```
Comment (yguedidi) [inline src/Facebook/Exceptions/FacebookResponseException.php]: must be removed

**Comment by yguedidi** [comment]
Thanks @tolbon!

**Pull request merged**